site/components/Help.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactElement } from "react";
+
+const Help = (): ReactElement => {
+    return (
+        <div>
+            <h2 className="govuk-heading-s">Help and Support</h2>
+            <p className="govuk-body">
+                If you are having problems, please contact the Create Disruption Service via this link:{" "}
+                <a href="/contact" className="govuk-link govuk-!-font-size-19">
```

Comment:
I think for contact, steve said to link to https://www.bus-data.dft.gov.uk/contact/

Reply:
done